tools/jit/gen_jit_dispatch.py

```diff
@@ -132,6 +132,13 @@ def from_ivalue(arg, value):
     return FROM_IVALUE[typ].format(value)
 
 
+CALL_UNBOXED_KERNEL = CodeTemplate("""\
+using FuncType = ${return_type} (${formals_types});
```

Comment:
Can this logic be put in a single-line function and not generated? It looks like that founction would be pretty simple

Reply:
This is supposed to replace CALL_NAMESPACE, CALL_METHOD, CALL_NAMESPACE_WITH_TENSOR_OPTIONS and CALL_METHOD_WITH_TENSOR_OPTIONS. That's why, instead of having those all call a common function, I chose to keep this separate.

Reply:
Sorry to be dense but I'm not sure I get the connection... why would replacing* those things make a common function undesirable?

*in the future

Reply:
ah I misunderstood the comment. Made it into a function now.